### PR TITLE
Removed the left from always being in the execute_aiming_action

### PR DESCRIPTION
**Pull request by jayRyan24**
People thought it was annoying. It will now preload the left per the skill.

### CODE REVIEW

**Comment by jayRyan24** [comment]
Don't merge yet, it's not working.

File: combat-trainer.lic [hunk 1]
```diff
@@ -2306,15 +2306,15 @@ class AttackProcess
       weapon_name = game_state.wield_offhand_weapon(skill)
 
       actions = if skill.include?('Thrown')
-                  %w[lob lob lob]
+                  %w['lob left' 'lob left' 'lob left']
```
Comment (rpherbig) [inline combat-trainer.lic]: 💡 The `%w` syntax doesn't work for multiple-word-phrases. I suspect you want `['lob left', 'lob left', 'lob left']`

**Comment by rpherbig** [comment]
❓ It looks like the only thing that functionally changes is the Tactics case, which would have been bob left/weave left/circle left but are now bob/weave/circle. Is that right?

**Comment by Sarvatt** [comment]
that looks right to me.

**Comment by jayRyan24** [comment]
Yes, the only thing really affected by this is the `tactics + left`. I tried single quotes and it failed, though I might have had another change which made it do so.

**Pull request merged**